gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -1874,11 +1918,16 @@ gum_exec_block_write_ret_transfer_code (GumExecBlock * block,
   gum_exec_block_close_prolog (block, gc);
   gum_exec_block_open_prolog (block, GUM_PROLOG_MINIMAL, gc);
 
+  gum_exec_ctx_load_real_register_into (block->ctx, ARM64_REG_X16, ret_reg, 0,
+      gc);
+
   gum_arm64_writer_put_push_all_x_registers (cw);
+  gum_arm64_writer_put_push_all_q_registers (cw);
```

Comment:
We don't save registers here on x86, because that's what the opened prolog has already done. Is this a bug?